cmd/th/main.go

```diff
@@ -0,0 +1,156 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"log"
+	"sync"
+	"time"
+
+	kafka "github.com/conduitio/conduit-connector-kafka"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+)
+
+const (
+	recordCount     = 1_000_000
+	checkpointCount = 10
+)
+
+func main() {
```

Comment:
This is a utility I used to test the throughput of the connector. It could be extracted and made into a connector performance testing tool, since it's connector agnostic.